### PR TITLE
Fix issue 29587: remove unhelpful link to Unicode

### LINKED ISSUE #29587: Link to Unicode
### MDN URL

https://developer.mozilla.org/en-US/docs/Glossary/I18N

### What specific section or headline is this issue about?

_No response_

### What information was incorrect, unhelpful, or incomplete?

The link to [Unicode](https://www.techtarget.com/whatis/definition/Unicode) seems to require signing in to see it.

### What did you expect to see?

Users don’t need to sign in to see content, and preferably the content should be written by i18n/Unicode experts.

### Do you have any supporting links, references, or citations?

Maybe it can link to https://www.w3.org/TR/i18n-glossary/#dfn-unicode ?

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/glossary/i18n`
* MDN URL: https://developer.mozilla.org/en-US/docs/Glossary/I18N
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/glossary/i18n/index.md
* Last commit: https://github.com/mdn/content/commit/ada5fa5ef15eadd44b549ecf906423b4a2092f34
* Document last modified: 2023-06-08T13:19:16.000Z

</details>

### PR DESCRIPTION
**Pull request by wbamberg**
Fixes https://github.com/mdn/content/issues/29587.

I found that we had two pages for internationalization:
- https://developer.mozilla.org/en-US/docs/Glossary/I18N
- https://developer.mozilla.org/en-US/docs/Glossary/Internationalization_and_localization (which despite the slug, only covers internationalization - localization gets https://developer.mozilla.org/en-US/docs/Glossary/Localization, although it is not linked).

So I wrote a new page at https://developer.mozilla.org/en-US/docs/Glossary/Internationalization, deleted the other two pages and redirected them here.

I didn't in the end link to https://developer.mozilla.org/en-US/docs/Glossary/unicode, although perhaps I should have. But I don't think that's a great page either - Unicode is more than a character set, it includes work on other aspects of i18n like date/time formatting and currencies. So I linked to https://home.unicode.org/ instead, which has great resources linked.

### CODE REVIEW

File: files/en-us/glossary/internationalization/index.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: Internationalization
```
Comment (bsmth) [inline files/en-us/glossary/internationalization/index.md:2]: What do you think about adding `i18n` into the title so we have site search hits for it?

```suggestion
title: Internationalization (i18n)
```

File: files/en-us/glossary/internationalization/index.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+**Internationalization**, often shortened to "i18n", is the practice of designing a system in such a way that it can easily be adapted for different target audiences, that may vary in region, language, or culture.
+
+The complementary process of adapting a system for a specific target audience is called {{glossary("Localization")}}.
+
+Among other things, internationalization requires support for:
+
+- different writing systems
+- different units of measure (currency, °C/°F, km/miles, etc.)
+- different time and date formats
+- different keyboard layouts
```
Comment (bsmth) [inline files/en-us/glossary/internationalization/index.md:18]: What do you think?

```suggestion
Among other things, internationalization covers adapting to differences in:

- writing systems
- units of measure (currency, °C/°F, km/miles, etc.)
- time and date formats
- keyboard layouts
```

**Review by bsmth — APPROVED** [review]
LGTM with some non-blocking suggestions for your consideration!

Thank you :)

**Comment by bsmth** [comment]
> Unicode is more than a character set, it includes work on other aspects of i18n like date/time formatting and currencies.

This is good to highlight

File: files/en-us/glossary/internationalization/index.md (hunk 1 above)
Comment (wbamberg) [reply files/en-us/glossary/internationalization/index.md:2]: Yeah, that's a good idea.

**Pull request merged**